iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java

```diff
@@ -19,11 +20,6 @@ public void testMainWorkflow() throws Exception {
 	}
 
     @Test
-	public void testMainSQLiteWorkflow() throws Exception{
```

Comment:
It turned out not all tests were failing due to the problem which was patched by Ioannis. 

One test was utilizing predefined sqlite db which become obsolete after minor change in projects model conducted recenty. In order to fix this test we would need to manually recreate db from scratch while in all other tests db was generated on the fly. I just found this test pointless.